src/drt/src/dr/FlexDR.cpp

```diff
@@ -746,7 +746,8 @@ void FlexDR::searchRepair(const SearchRepairArgs& args)
     cout << flush;
   }
   end();
-  if (logger_->debugCheck(DRT, "autotuner", 1)
+  if ((DRC_RPT_ITER_STEP > 0 && iter % DRC_RPT_ITER_STEP == 0)
```

Comment:
delete previous flow's reports before starting current flow. The maximum number of iterations are 64.

Or leave that to ORFS?

Reply:
I tend to prefer the tool doesn't remove user files it isn't overwriting.  @gadfort any thoughts from the SiC perspective on tool vs flow?

Some designs generate a lot of early iteration drcs.  bp_single is ~400K.  I'm not clear having it on by default is always a benefit.  It is easy for you to add it to your project settings.

Reply:
My concern is not about me, but about training when having to learn the tool. Is there no acceptable default behavior for ORFS to generate these files?

Modifying detailed routing options and global routing options is tedious in ORFS. I think for detailed routing it requires the user to run the flow once, then find the options in the log, then manually modify the entire set of options. This is exactly the sort of thing that I want to avoid for OpenROAD novices...

Reply:
I would suggest having it off by default, 8t would be relatively easy to add a parameter to SC to allow for users to turn it on or off themselves. SC takes care of cleaning up the runtime directories so no need to remove the files ahead.